solr/benchmark/README.md

```diff
@@ -1,339 +1,423 @@
 provided by the JMH project.
 
-JMH is highly configurable and users are encouraged to look through the samples for suggestions on what options are
-available. A good tutorial for using JMH can be
+JMH is highly configurable, and users are encouraged to look through the samples
+for exposure around what options are available. A good tutorial for learning JMH basics is
 found [here](http://tutorials.jenkov.com/java-performance/jmh.html#return-value-from-benchmark-method)
 
-Many Solr JMH benchmarks are actually closer to a full integration benchmark in that they run a single action against a
-full Solr mini cluster.
-
-See [org.apache.solr.bench.index.CloudIndexing](https://github.com/apache/solr/blob/main/solr/benchmark/src/java/org/apache/solr/bench/index/CloudIndexing.java)
-for an example of this.
-
-### SolrCloud MiniCluster Benchmark Setup
-
-#### MiniCluster Setup
-
-- CloudIndexing.java
-
-```java
-    @Setup(Level.Trial)
-    public void doSetup(MiniClusterState.MiniClusterBenchState miniClusterState) throws Exception {
-      System.setProperty("mergePolicyFactory", "org.apache.solr.index.NoMergePolicyFactory");
-      miniClusterState.startMiniCluster(nodeCount);
-      miniClusterState.createCollection(COLLECTION, numShards, numReplicas);
-    }
-```
-
-#### MiniCluster Metrics
-
-After every iteration, the metrics collected by Solr will be dumped to the build/work/metrics-results folder. You can
-disable metrics collection using the metricsEnabled method of the MiniClusterState, in which case the same output files
-will be dumped, but the values will all be 0/null.
-
-### Benchmark Repeatability
-
-Indexes created for the benchmarks often involve randomness when generating terms, term length and number of terms in a
-field. In order to make benchmarks repeatable, a static seed is used for randoms. This allows for generating varying
-data while ensuring that data is consistent across runs.
-
-You can vary that seed by setting a system property to explore a wider range of variation in the benchmark:
-
-`-jvmArgsAppend -Dsolr.bench.seed=6624420638116043983`
-
-The seed used for a given benchmark run will be printed out near the top of the output.
-
-> --> benchmark random seed: 6624420638116043983
-
-You can also specify where to place the mini-cluster with a system property:
+## Continued Documentation
 
-`-jvmArgsAppend -DminiClusterBaseDir=/benchmark-data/mini-cluster`
+### 📚 Profilers
 
-In this case, new data will not be generated for the benchmark, even if you change parameters. The use case for this if
-you are running a query based benchmark and want to create a large index for testing and reuse (say hundreds of GB's).
-Be aware that with this system property set, that same mini-cluster will be reused for any benchmarks run, regardless of
-if that makes sense or not.
+- 📒 [docs/jmh-profilers.md](docs/docs/jmh-profilers.md)
+- 📒 [docs/jmh-profilers-setup.md](docs/docs/jmh-profilers-setup.md)
```

Comment:
is docs/docs correct for these two links?

Reply:
It's not - I fixed the links in a recent patch.